predicators/approaches/bilevel_planning_approach.py

```diff
@@ -125,19 +125,14 @@ def _get_current_predicates(self) -> Set[Predicate]:
         return self._initial_predicates
 
     def get_option_model(self) -> _OptionModelBase:
-        """For ONLY an oracle approach or BEHAVIOR, we allow the user to get
-        the current option model."""
-        assert self.get_name() == "oracle" or CFG.env == "behavior"
+        """For ONLY an oracle approach, we allow the user to get the current
+        option model."""
+        assert self.get_name() == "oracle"
         return self._option_model
 
     def get_last_plan(self) -> List[_Option]:
-        """For ONLY BEHAVIOR, we allow the user to get the plan that was most
```

Comment:
looks like this comment was outdated, since we are using the function in `predicators/datasets/demo_only.py` to collect demos, so good call on keeping it